models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
```

Comment:
Where/how is the identifier captured?

Reply:
The patient registration app doesnt record identifiers. May you clarify?